config.toml

```
@@ -442,7 +442,7 @@ language_alternatives = ["en"]
 [languages.hi]
 title = "Kubernetes"
 description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
-languageName = "हिन्दी"
+languageName = "हिनदी (Hindi)"  
```

Just a small nit, but otherwise LGTM:
```suggestion
languageName = "हिन्दी (Hindi)"  
```